Dungeons/src/main/java/dungeons/Game.java

```diff
@@ -0,0 +1,57 @@
+package dungeons;
+
+import dungeons.factory.RoomFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Game {
+    private Player player;
+    private Room currentRoom;
+    private static final Map<String, Integer> directionMap;
+
+    static {
+        directionMap = new HashMap<>();
+        directionMap.put("left", 1);
+        directionMap.put("right", 2);
+        directionMap.put("forwards", 3);
+        directionMap.put("backwards", 4);
+    }
+
+    public Game() {
+        this.player = new Player();
+        this.currentRoom = RoomFactory.createRoom();
+        this.currentRoom.generateAdjacentRooms();
+    }
+
+    public void startGame() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            System.out.println("\nIn which direction you want to go? (left, right, forwards, backwards)");
+            String choice = scanner.next().toLowerCase();
+            Integer numericChoice = directionMap.get(choice);
+
+            if (numericChoice != null && numericChoice >= 1 && numericChoice <= 4) {
```

Comment:
take out the condition to a more variable with a more readable name

Reply:
did